tenbilac/net.py

```diff
@@ -1,18 +1,15 @@
 """
 This is Tenbilac!
-Net represent a "simple" network. See WNet if you're looking for weight predictions.
+Net represent a "simple" network. See WNet if you're looking for weight(ed) predictions.
```

Comment:
Note that the WNet experiment is kind of deprecated. I currently guess that we will use different approaches to get weights. So it's not a priority to deal with those wnets.